snuba/consumers/consumer_builder.py

```diff
@@ -221,7 +261,11 @@ def __build_consumer(
     def __build_streaming_strategy_factory(
         self,
     ) -> ProcessingStrategyFactory[KafkaPayload]:
-        table_writer = self.storage.get_table_writer()
+        # this assumes single storage for now
```

Comment:
This function is quite different currently between the two implementations. What do you think about leaving it unchanged for now and building an alternativve `__build_streaming_multistorage_strategy_factory` alongside it?

Reply:
I think that's a good idea. It would probably be one of my next steps.